extension.js

```diff
@@ -105,17 +122,27 @@ class MprisLabel extends PanelMenu.Button {
 	_updateTrayPosition(){
 		const EXTENSION_PLACE = this.settings.get_string('extension-place');
 		const EXTENSION_INDEX = this.settings.get_int('extension-index');
+		const SHOW_CONTROL = this.settings.get_boolean('show-control');
 
 		this.container.get_parent().remove_child(this.container);
 
 		if(EXTENSION_PLACE == "left"){
 			Main.panel._leftBox.insert_child_at_index(this.container, EXTENSION_INDEX);
+			if(SHOW_CONTROL){
+				Main.panel._leftBox.insert_child_at_index(this.buttonContainer, EXTENSION_INDEX + 1);
+			}
 		}
 		else if(EXTENSION_PLACE == "center"){
 			Main.panel._centerBox.insert_child_at_index(this.container, EXTENSION_INDEX);
+			if(SHOW_CONTROL){
+				Main.panel._leftBox.insert_child_at_index(this.buttonContainer, EXTENSION_INDEX + 1);
```

Comment:
wrong location should be `Main.panel._centerBox.insert_child_at_index(this.buttonContainer, EXTENSION_INDEX + 1);`

Reply:
fixed

Reply:
great. I think that you might need to mark this as `resolved` yourself.